lib/lambda/update/updatePackage.ts

```diff
@@ -0,0 +1,232 @@
+import { response } from "libs/handler-lib";
+import { APIGatewayEvent } from "aws-lambda";
+import { getPackage } from "libs/api/package";
+import { produceMessage } from "libs/api/kafka";
+import { ItemResult } from "shared-types/opensearch/main";
+import { getPackageType } from "./getPackageType";
+import { events } from "lib/packages/shared-types";
+import { z } from "zod";
+
+const sendDeleteMessage = async (packageId: string) => {
+  const topicName = process.env.topicName as string;
+  if (!topicName) {
+    throw new Error("Topic name is not defined");
+  }
+  await produceMessage(
+    topicName,
```

Comment:
Out of scope, for sure, but I think moving `topicName` to be handled in `produceMessage` would simplify things quite a bit here since it'll always need to be passed and it's always an `env` variable. After searching, `produceMessage` is only used once, so there isn't a lot of adjusting needed to make such a change

Reply:
Totally agree, not sure why it wasn't handled there initially